rag_assistant/pages/0_Chat_with_Docs.py

```diff
@@ -14,41 +13,40 @@
 from langchain_core.prompts import ChatPromptTemplate, MessagesPlaceholder
 from langchain_core.runnables.history import RunnableWithMessageHistory
 from langsmith import traceable
-
 from shared.llm_facade import get_conversation_starters
 from utils.auth import check_password
 from utils.constants import Metadata
 from utils.utilsdoc import get_store, extract_unique_name
 from utils.config_loader import load_config
 from streamlit_feedback import streamlit_feedback
 import logging
-
 from streamlit.runtime.scriptrunner import add_script_run_ctx
 from streamlit.runtime.scriptrunner.script_run_context import get_script_run_ctx
-
 from utils.utilsllm import load_model
+import time
 
 load_dotenv(find_dotenv())
 openai_api_key = os.getenv('OPENAI_API_KEY')
 
 # set logging
-logger = logging.getLogger('AI_assistant_feedback')
+logger = logging.getLogger('')
 logger.setLevel(logging.INFO)
 
-# Check if the directory exists, if not create it
 log_dir = "logs"
 if not os.path.exists(log_dir):
     os.makedirs(log_dir)
 
-# Create a file handler for the logger
-handler = logging.FileHandler(os.path.join(log_dir, 'feedback.log'))
-handler.setLevel(logging.INFO)
+# handler = logging.FileHandler(os.path.join(log_dir, 'feedback.log'))
+# handler.setLevel(logging.INFO)
 
-# Create a logging format
-formatter = logging.Formatter('%(asctime)s -  %(message)s')
-handler.setFormatter(formatter) # Add the formatter to the handler  
+# formatter = logging.Formatter('%(asctime)s - %(message)s')
+# handler.setFormatter(formatter)
 
-# Add the handler to the logger
+logger.handlers.clear()
```

Comment:
J'aurais essayé de vérifier si le Handler est déjà présent dans le logger.handlers plutôt que de faire un clear, mais ça fait le taf

Reply:
En utilisant cette méthode:

if not logger.handlers:
    handler = logging.FileHandler(os.path.join(log_dir, 'feedback.log'))
    handler.setLevel(logging.INFO)
    formatter = logging.Formatter('%(asctime)s - %(message)s')
    handler.setFormatter(formatter)
    logger.addHandler(handler)